### PR TITLE
Accessory armor vests! 

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]: 
Adds in many different types of vest for each department to help people better survive in there 100% safe work!
Adds in a new node having more advance protection for sec!
List of vest added
Illegal Vest -Admin only ATM
Biohazard resistant vest
Bomb resistant vest
Fire resistant vest
Lava Land Vest
Miner Vest
Ballistic vest -Civ
Close Quarter Combat Vest -Civ
Military grade physal armor vest - Sec
Most armor dosn't do all that much for more combat, everything is less then or the same to bone talisman, in the combat
[why]: 
Accessory slot is never used actively unless your a miner, lets change that!

### CODE REVIEW

File: code/modules/clothing/under/accessores/armor.dm [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/obj/item/clothing/accessory/synd_vest
+	name = "Illegal Vest"
+	desc = "A illegal vest with a mix of diffent metals and plastics."
```
Comment (Poojawa) [inline code/modules/clothing/under/accessores/armor.dm]: diffent

File: code/modules/clothing/under/accessores/armor.dm [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 10, "bullet" = 15, "laser" = 15,"energy" = 5, "bomb" = 15, "bio" = 5, "rad" = 10, "fire" = 10, "acid" = 20)
+
+/obj/item/clothing/accessory/bio_vest
+	name = "Biohazard resistant vest."
+	desc = "Biohazard resistant vest that is easily attached to a jumpsuit."
+	icon_state = "waistcoat"
```
Comment (Poojawa) [inline code/modules/clothing/under/accessores/armor.dm:11]: copypasta out the ass btw

File: code/modules/clothing/under/accessores/armor.dm [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+	item_state = "waistcoat"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 40, "rad" = 0, "fire" = 0, "acid" = 10)
+
+/obj/item/clothing/accessory/bomb_vest
+	name = "Bomb resistant vest."
+	desc = "A heavy vest that can be attached to a jumpsuit to help protect against a blast."
```
Comment (Poojawa) [inline code/modules/clothing/under/accessores/armor.dm:17]: if it's heavy it should cause some slow down penalties

File: code/modules/clothing/under/accessores/armor.dm [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+	item_state = "waistcoat"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 40, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+
+
+/obj/item/clothing/accessory/fire_vest
+	name = "Fire resistant vest."
+	desc = "A vest that can be placed over normal clothing to help protect against fire.."
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 0)
```
Comment (Poojawa) [inline code/modules/clothing/under/accessores/armor.dm:28]: 50% fire damage nerfing. lol

File: code/modules/clothing/under/accessores/armor.dm [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+	armor = list("melee" = 15, "bullet" = 5, "laser" = 0,"energy" = 0, "bomb" = 15, "bio" = 5, "rad" = 0, "fire" = 0, "acid" = 15)
+
+/obj/item/clothing/accessory/sec_ballistic_vest
+	name = "Military grade bullet armor vest."
+	desc = "A well made vest that helps protect against bullets. In the space days its still as basic as it always was."
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 5, "bullet" = 15, "laser" = 0,"energy" = 0, "bomb" = 15, "bio" = 5, "rad" = 0, "fire" = 0, "acid" = 15)
+
+/obj/item/clothing/accessory/engery_vest
```
Comment (Poojawa) [inline code/modules/clothing/under/accessores/armor.dm:72]: engery_vest

File: code/modules/clothing/under/accessores/armor.dm [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+	name = "Laser Dispersion Vest.."
```
Comment (Poojawa) [inline code/modules/clothing/under/accessores/armor.dm]: Vest..

File: code/modules/clothing/under/accessores/armor.dm [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 5,"energy" = 25, "bomb" = 0, "bio" = 15, "rad" = 10, "fire" = 10, "acid" = 5)
+
+/obj/item/clothing/accessory/rad_vest
+	name = "Radiation vest."
+	desc = "Heavy plastic padding with some plasma to greatly reduce radiation."
+	icon_state = "waistcoat"
+	item_state = "waistcoat"
+	armor = list("melee" = 0, "bullet" = 0, "laser" = 0,"energy" = 15, "bomb" = 15, "bio" = 5, "rad" = 50, "fire" = -50, "acid" = 25)
```
Comment (Poojawa) [inline code/modules/clothing/under/accessores/armor.dm:91]: "fire" = -50

File: code/modules/research/designs/vest_designs.dm [hunk 8]
```diff
@@ -0,0 +1,120 @@
...
+	desc = "Bio Hazer resistent vest that is easly attached to a jumpsuit."
+	id = "bio_vest"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 1500, MAT_GLASS = 1000, MAT_GOLD = 1750)
+	build_path = /obj/item/clothing/accessory/bio_vest
+	category = list("Misc")
+	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_MEDICAL
+
+/datum/design/bomb_vest
+	name = "Bomb resitsent vest."
```
Comment (Poojawa) [inline code/modules/research/designs/vest_designs.dm]: resitsent

File: code/modules/research/designs/vest_designs.dm [hunk 9]
```diff
@@ -0,0 +1,120 @@
+/datum/design/bio_vest
+	name = "Bio Hazer resistent vest"
```
Comment (Poojawa) [inline code/modules/research/designs/vest_designs.dm]: Hazer

File: code/modules/research/designs/vest_designs.dm [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+	id = "melee_vest"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 7500, MAT_GLASS = 5500, MAT_SILVER = 2250, MAT_TITANIUM = 5500)
+	build_path = /obj/item/clothing/accessory/melee_vest
+	category = list("Misc")
+	departmental_flags = DEPARTMENTAL_FLAG_SERVICE
+
+/datum/design/rad_vest
+	name = "Radiation vest."
+	desc = "Heavy plastic padding with some plasma to greaty redeuce radition.."
```
Comment (Poojawa) [inline code/modules/research/designs/vest_designs.dm:74]: redeuce

File: code/modules/research/designs/vest_designs.dm [hunk 11]
```diff
@@ -0,0 +1,120 @@
...
+	id = "rad_vest"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 5500, MAT_GLASS = 2500, MAT_SILVER = 2250, MAT_PLASMA = 5000, MAT_PLASTIC = 3500)
+	build_path = /obj/item/clothing/accessory/rad_vest
+	category = list("Misc")
+	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_ENGINEERING
+
+/datum/design/sec_melee_vest
+	name = "Millitart grade physal armor vest.."
```
Comment (Poojawa) [inline code/modules/research/designs/vest_designs.dm]: physal

File: code/modules/research/designs/vest_designs.dm [hunk 12]
```diff
@@ -0,0 +1,120 @@
...
+	desc = "A well made vest that helps protect againts phycal harm. NT still cuts corrners when they can and uses less then optimal matitarls."
```
Comment (Poojawa) [inline code/modules/research/designs/vest_designs.dm:84]: matitarls

File: code/modules/research/designs/vest_designs.dm [hunk 13]
```diff
@@ -0,0 +1,120 @@
...
+	id = "sec_melee_vest"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 7500, MAT_GLASS = 5500, MAT_GOLD = 2250, MAT_TITANIUM = 5500, MAT_DIAMOND = 4000)
+	build_path = /obj/item/clothing/accessory/sec_melee_vest
+	category = list("Misc")
+	departmental_flags =  DEPARTMENTAL_FLAG_SECURITY
+
+/datum/design/sec_ballistic_vest
+	name = "Millitart grade physal armor vest."
```
Comment (Poojawa) [inline code/modules/research/designs/vest_designs.dm]: Millitart

File: code/modules/research/designs/vest_designs.dm [hunk 14]
```diff
@@ -0,0 +1,120 @@
...
+	desc = "A well made vest that helps protect againts bullets. In the space day its still as basic as it always was."
+	id = "sec_ballistic_vest"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 7500, MAT_GLASS = 5500, MAT_GOLD = 2250, MAT_TITANIUM = 5500, MAT_DIAMOND = 4000)
+	build_path = /obj/item/clothing/accessory/sec_ballistic_vest
+	category = list("Misc")
+	departmental_flags =  DEPARTMENTAL_FLAG_SECURITY
+
+/datum/design/power_vest
+	name = "Engery protection padding."
```
Comment (Poojawa) [inline code/modules/research/designs/vest_designs.dm]: Engery

File: code/modules/research/designs/vest_designs.dm [hunk 15]
```diff
@@ -0,0 +1,120 @@
...
+	desc = "Padding used on suits to try to prevent shocks form being as lethal."
+	id = "power_vest"
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 7500, MAT_GLASS = 5500, MAT_GOLD = 7250, MAT_SILVER = 7250)
+	build_path = /obj/item/clothing/accessory/power_vest
+	category = list("Misc")
+	departmental_flags =  DEPARTMENTAL_FLAG_ENGINEERING
+
+/datum/design/engery_vest
+	name = "Laser disbushen vest."
```
Comment (Poojawa) [inline code/modules/research/designs/vest_designs.dm]: disbushen

File: code/modules/research/techweb/all_nodes.dm [hunk 16]
```diff
@@ -599,6 +599,15 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
 	export_price = 5000
 
+/datum/techweb_node/sec_vests
+	id = "sec_vests"
+	display_name = "Military Vests"
+	description = "High grade protection vests for Civ and Military presents.."
+	prereq_ids = list("adv_weaponry", "ballistic_weapons", "adv_engi")
+	design_ids = list("ballistic_vest", "melee_vest", "sec_ballistic_vest", "sec_melee_vest", "engery_vest",  "engery_vest")
```
Comment (Poojawa) [inline code/modules/research/techweb/all_nodes.dm]: engery

also it's listed twice.

all of these have the exact same icon, so you can't differentiate from them.

**Review by Poojawa — CHANGES_REQUESTED** [review]
Please please please spell check everything

also remove the '.' from names

**Comment by deathride58** [comment]
Conflicts

**Comment by Trilbyspaceclone** [comment]
*When will I learn*...

**Pull request closed without merging**